client/containers/Pub/SpubHeader/SpubHeaderToolBar/spubHeaderToolbar.scss

```diff
@@ -0,0 +1,40 @@
+@import 'styles/variables.scss';
+
+.spubheader-toolbar {
+	display: flex;
+	justify-content: center;
+	align-items: baseline;
+	column-gap: 340px;
+	.tabs.spub-header-component .bp3-tab-list {
+		margin: 0px;
+	}
+	.spub-header-component .bp3-tab-list .bp3-tab.inactive {
+		opacity: 0.45;
+		color: var(--community-accent-dark);
+	}
+}
+
+.gridParent {
+	&.container {
+		max-width: 100%;
+	}
+}
+
+.status {
+	display: flex;
+}
+
+.status-text {
+	text-transform: capitalize;
+}
+
+.submission-button {
+	width: 119px;
+	height: 30px;
```

Comment:
I think you can drop all this and use the `minimal` prop on the button instead.

Also, please set `width: 120px`. I know it is exactly `119px` in the mockup, but this number is driving me crazy!

Reply:
🙃🙃🙃